utils/jwe.ts

```diff
@@ -0,0 +1,55 @@
+import * as E from "fp-ts/Either";
+import * as TE from "fp-ts/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import * as jose from "jose";
+
+import { addSeconds } from "date-fns";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { Second } from "@pagopa/ts-commons/lib/units";
+
+/**
+ * JWE Generation
+ */
+export type GetGenerateJWE = <T extends jose.JWTPayload>(
+  issuer: NonEmptyString,
+  jweKey: NonEmptyString
+) => (payload: T, ttl: Second) => TE.TaskEither<Error, NonEmptyString>;
+
+const secondsFromEpoch = (secondsToAdd: number): Second => {
+  const newDate = addSeconds(new Date(), secondsToAdd);
+  return Math.floor(newDate.getTime() / 1000) as Second;
+};
```

Comment:
What if we use an existing utils to calculate the timestamp in seconds, for example using [getUnixTime](https://date-fns.org/v2.30.0/docs/getUnixTime)
```suggestion
const secondsFromEpoch = (secondsToAdd: number): Second =>
  getUnixTime(addSeconds(new Date(), secondsToAdd));
```